### ISSUE TITLE
Ролевая модель для пользователей

### ISSUE THREAD

**Issue by hodkovdd** [sedovalx/taxi#96]
Функционал в виде форм и кнопок должен быть доступен определенным пользователям в соответствии с их ролями. Конкретно что и кому будет описано здесь позже.

**Comment by sedovalx** [comment]
Входит в план-минимум #95

**Comment by kirzas** [comment]
@sedovalx 
Как вообщее ее делать?
Нормально,если я на каждый объект в шаблоне поставлю if, и там буду роль проверять?
в этом случае по url все равно можно будет выполнить действия.

**Comment by sedovalx** [comment]
Да, для клиента так и нужно. Только это должно быть сделано в едином стиле во всех шаблонах и контроллерах. Например в базовом контроллере можно объявить метод 

```
hasRole(string): Boolean
```

Который возвращает true, если текущий пользователь входит в роль с переданным именем. Так же можно добавить туда же конкретные методы типа

```
isAdmin = () => hasRole('admin')
```

Тогда в шаблонах можно будет их использовать при отображении кнопок. 

Да, через uri всё равно можно будет выполнить действие. Поэтому авторизация должна быть не только на клиенте, но и на сервере. На сервере в первую очередь, кстати. В какой-то задаче я это уже пояснял.

**Comment by sedovalx** [comment]
Смотри [wiki](https://github.com/sedovalx/taxi/wiki/%D0%91%D0%B0%D0%B7%D0%B0-%D0%B7%D0%BD%D0%B0%D0%BD%D0%B8%D0%B9.-%D0%91%D0%B5%D0%B7%D0%BE%D0%BF%D0%B0%D1%81%D0%BD%D0%BE%D1%81%D1%82%D1%8C.-Auth#%D0%9A%D0%BB%D0%B8%D0%B5%D0%BD%D1%82%D1%81%D0%BA%D0%B0%D1%8F-%D0%B0%D1%83%D1%82%D0%B5%D0%BD%D1%82%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%86%D0%B8%D1%8F)
